src/dappsExplorer/DappFilterChip.tsx

```diff
@@ -17,8 +17,8 @@ interface DappFilterChip {
 
 function DappFilterChip({ filterId, filterName, isSelected, onPress, style }: DappFilterChip) {
   const filterPress = () => {
+    isSelected ? onPress('all') : onPress(filterId)
```

Comment:
this is unnecessary right? since the `filterPress` callback that's passed in already does this check